pkg/ingester/ingester_v2_test.go

```diff
@@ -1858,12 +1858,12 @@ func TestIngester_flushing(t *testing.T) {
 				i.FlushHandler(httptest.NewRecorder(), httptest.NewRequest("POST", "/flush", nil))
 
 				// Flush handler only triggers compactions, but doesn't wait for them to finish. Let's wait for a moment, and then verify.
-				test.Poll(t, 1*time.Second, true, func() interface{} {
+				test.Poll(t, 5*time.Second, uint64(0), func() interface{} {
```

Comment:
This test was flaky again on my local machine, so I had:
1. Increase polling timeout from 1s to 5s
2. Compared on the actual number, so we can see what's the value of `NumSeries()` when it fails